google/cloud/security/common/gcp_type/firewall_rule.py

```diff
@@ -178,12 +185,13 @@ def __eq__(self, other):
         """
         return (self.direction == other.direction and
                 self.network == other.network and
-                self.source_tags == other.source_tags and
-                self.target_tags == other.target_tags and
+                self._source_tags == other._source_tags and
+                self._target_tags == other._target_tags and
                 self.source_ranges == other.source_ranges and
                 self.destination_ranges == other.destination_ranges and
                 self.firewall_action == other.firewall_action)
 
+    # pylint: disable=protected-access
```

Comment:
At the end of the function add another comment to re-enable protected-access checks.

\# pylint: enable=protected-access